src/pages/page/dashboard/dashboard.jsx

```diff
@@ -0,0 +1,16 @@
+import { NavLink } from 'react-router-dom';
```

Comment:
You should use `pages/dashboard/index.jsx` route for the specific page. As a result, imports will be much shorter. The same issue is present on the rest of the pages.